app/src/main/java/com/android/sample/ui/createRecipe/RecipeListInstructionsScreen.kt

```diff
@@ -102,12 +107,30 @@ fun RecipeListInstructionsContent(
         modifier = Modifier.height(BIG_PADDING.dp).testTag(RECIPE_LIST_INSTRUCTIONS_SCREEN_SPACER1))
 
     // This text represents the name of the recipe
-    Text(
-        modifier = Modifier.testTag(RECIPE_NAME_TEXT),
-        text = createRecipeViewModel.getRecipeName(),
-        style = MaterialTheme.typography.titleMedium,
-        color = MaterialTheme.colorScheme.onPrimary,
-    )
+    Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
+      Text(
+          modifier = Modifier.testTag(RECIPE_NAME_TEXT),
+          text =
+              if (createRecipeViewModel.getRecipeName().length > RECIPE_NAME_MAX_CHAR)
+                  createRecipeViewModel.getRecipeName().take(RECIPE_NAME_MAX_CHAR) + "..."
```

Comment:
The addition of a character limit for the recipe name with ellipsis is a nice improvement for UI consistency on smaller screens, Great JOb

Reply:
Thanks dude !